src/main/java/com/google/sps/data/LoginStatus.java

```diff
@@ -22,12 +22,16 @@
     private String url;
     private String userId;
     private String userEmail;
+    private boolean tutor;
+    private boolean student;
 
-    public LoginStatus(boolean isLoggedIn, boolean needsToRegister, String url, String userId, String userEmail) {
+    public LoginStatus(boolean isLoggedIn, boolean needsToRegister, String url, String userId, String userEmail, boolean tutor, boolean student) {
```

Comment:
I think we really only need one of these booleans here. For example, if we kept just the `tutor` boolean, we could check if the user is a tutor by checking if `tutor` is true or we could check if the user is a student by checking if `tutor` is false.

Reply:
The thing is that when we start having the possibility of a user being both a tutor and a student, we would have to have the two different fields. Otherwise, someone would have to create two different accounts, one for tutoring and one for learning, which I do not think is the best approach. Right now, I was envisioning having a tab (similar to the one that makes the distinction between book and tutors in the search area) to display the information of a user's learning activities and a tab for their tutoring activities. Of course, if the user is only a tutor or a student then that tab would not be displayed.

Reply:
Oh yeah, that makes sense!

Reply:
I just go rid of this now because I'm using role.